src/components/DataController/DataController.js

```diff
@@ -180,6 +193,14 @@ var DataController = React.createClass({
               <Col lg={4} lgPush={8} md={6} mdPush={6} sm={6} smPush={6}>
                 <TimeOfYearSelector onChange={this.updateProjChangeTimeOfYear} />
               </Col>
+              <Col>
+                <div>
+                  <ControlLabel>Download Data</ControlLabel>
+                  <br/>
```

Comment:
As above. If you go the CSS route, your selector should capture all these cases. There may even be an existing Bootstrap class you can apply.